src/news/release-8.3.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Prism Launcher Release 8.3, now available"
+description: Also try Terraria
+date: 2024-03-03
+release_version: "8.3"
+minimum_macos_version: 11.0.0
+macos_file_extension: zip
+macos_signature: u4nVbCdMLaPkIYma3VBONC9Nxwb+Nsad4fBqJ8Un9HcBCYSrRjrsja7A+cdWO/mrP8JUt6EyhGgLpGYDT2vZAA==
+tags:
+  - release
+---
+
+Hello there!
+
+We have accumulated a few changes and fixes in the past few weeks.
+This release fixes problems with some non-latin instance names.
```

Comment:
I don't think that's bigger than mod info for neoforge 1.20.5 and curseforge export fix